package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
```

Comment:
- [ ] The test script can be deleted since it's not being used

- [ ] Both scripts are not informing in which mode Webpack should run. Providing the mode configuration option tells Webpack to use its built-in optimizations accordingly.
```javascript
"scripts": {
    "build": "webpack --mode production",
    "watch": "webpack --mode development --watch",
  },
```